.rhiza/tests/api/test_makefile_api.py

```
@@ -91,38 +90,22 @@ def setup_api_env(logger, root, tmp_path: Path):
         os.chdir(old_cwd)
 
 
-def run_make(args: list[str] | None = None, dry_run: bool = True) -> subprocess.CompletedProcess:
-    """Run make in the current directory."""
-    cmd = [MAKE]
-    if dry_run:
-        cmd.append("-n")
-    if args:
-        cmd.extend(args)
+# Import run_make from local conftest
+from .conftest import run_make  # noqa: E402
```

Ruff (0.14.14)

[warning] Unused `noqa` directive (non-enabled: `E402`)

Remove unused `noqa` directive

(RUF100)